src/common/proc.c

```diff
@@ -94,34 +91,36 @@ crt_proc_prop_entries(crt_proc_t proc, crt_proc_op_t opc, daos_prop_t *prop)
 		    entry->dpe_type == DAOS_PROP_CO_OWNER ||
 		    entry->dpe_type == DAOS_PROP_PO_OWNER_GROUP ||
 		    entry->dpe_type == DAOS_PROP_CO_OWNER_GROUP) {
-			rc = crt_proc_d_string_t(proc, &entry->dpe_str);
+			rc = crt_proc_d_string_t(proc, proc_op,
+						 &entry->dpe_str);
 
 		} else if (entry->dpe_type == DAOS_PROP_PO_ACL ||
 			 entry->dpe_type == DAOS_PROP_CO_ACL) {
-			rc = crt_proc_struct_daos_acl(proc,
+			rc = crt_proc_struct_daos_acl(proc, proc_op,
 						      (struct daos_acl **)
 						      &entry->dpe_val_ptr);
 
 		} else if (entry->dpe_type == DAOS_PROP_PO_SVC_LIST) {
-			rc = crt_proc_d_rank_list_t(proc,
+			rc = crt_proc_d_rank_list_t(proc, proc_op,
 					(d_rank_list_t **)&entry->dpe_val_ptr);
 		} else if (entry->dpe_type == DAOS_PROP_CO_ROOTS) {
 			struct daos_prop_co_roots *roots;
 
-			if (opc == CRT_PROC_DECODE) {
+			if (proc_op == CRT_PROC_DECODE) {
```

Comment:
There's a macro for this isn't there?
```suggestion
			if (DECODING(proc_op)) {
```

Reply:
Unfortunately this is defined in obj_rpc.h only. Probably it can be moved into common header in next patch.

Reply:
Done.